public/data/html.json

```diff
@@ -0,0 +1,118 @@
+[
+  {
+    "language": "html",
+    "categoryName": "Basic Layouts",
+    "snippets": [
+      {
+        "title": "Sticky Header-Footer Layout",
+        "description": "Full-height layout with sticky header and footer, using modern viewport units and flexbox.",
+        "code": [
+          "<!DOCTYPE html>",
+          "<html style=\"margin: 0\">",
+          "  <head></head>",
+          "  <body>",
+          "    <div class=\"header\">header</div>",
+          "    <div class=\"body\">body/content</div>",
+          "    <div class=\"footer\">footer</div>",
+          "  </body>",
+          "  <style type=\"text/css\">",
```

Comment:
Could you please move the style tag inside the head element ? Like you did for the second example ?

Reply:
Its a pain to do anything afterwards but I did the thing.

We really need a piece of code to go to and from the JSON

Reply:
#39